_episodes/03-matplotlib.md

```diff
@@ -13,15 +13,28 @@ keypoints:
 ---
 
 ## Visualizing data
-The mathematician Richard Hamming once said, "The purpose of computing is insight, not numbers," and
+The mathematician Richard Hamming once said, "The purpose of computing is insight, not numbers", and
 the best way to develop insight is often to visualize data.  Visualization deserves an entire
 lecture of its own, but we can explore a few features of Python's `matplotlib` library here.  While
 there is no official plotting library, `matplotlib` is the _de facto_ standard.  First, we will
-import the `pyplot` module from `matplotlib` and use two of its functions to create and display a
+import the `pyplot` module from `matplotlib`, and then we will use two of its functions to create and display a
 [heat map]({{ page.root }}/reference.html#heat-map) of our data:
 
+> ## Important Note
+>
+> In the previous lesson we set the `data` variable and imported `numpy`. If you have not completed lesson `2. Analyzing Patient Data`,
```

Comment:
this entire repository is a lesson and it's made of individual episodes. So, we're talking about the previous episode. Let's update the wording and link to the previous episode. The proper way to link to an episode is described in the [lesson-example](https://carpentries.github.io/lesson-example/04-formatting/index.html#locations-and-names), and in our case it'll be
```md
[Analyzing Patient Data]({{ page.root }}{% link _episodes/02-numpy.md %})
```

@ldko, could you please comment on the wording here (if you have any suggestions how to improve this)?

Reply:
Perhaps something like one of the following:

> In the previous episode, [Analyzing Patient Data]({{ page.root }}{% link _episodes/02-numpy.md %}), we imported `numpy` and set the `data` variable. If you have not yet done this, you will need to run the following code before continuing:

or

> If you are not continuing from the previous episode, [Analyzing Patient Data]({{ page.root }}{% link _episodes/02-numpy.md %}), you will need to import the `numpy` library and set the `data` variable: